source/deployment/webrtc.rst

```diff
@@ -7,17 +7,65 @@ The feature is a working prototype for community development and not recommended
+This guide is aimed to set up Mattermost WebRTC with a Docker image, `available here <https://hub.docker.com/r/mattermost/webrtc/>`_  and a Janus server to act as the WebRTC gateway. 
 
-To set up the WebRTC server, you may either
+If you want to use a full `Janus Gateway <https://janus.conf.meetecho.com/>`_, please visit their `GitHub repo <https://github.com/meetecho/janus-gateway>`_ for detailed instructions. You may also optionally set up `Coturn <https://github.com/coturn/coturn/wiki>`_ for STUN and TURN servers for your Mattermost installation.
 
- - use a `Mattermost Docker container created for testing WebRTC <https://hub.docker.com/r/mattermost/webrtc/>`_
- - set up a `Janus server <https://github.com/meetecho/janus-gateway>`_ to act as the WebRTC gateway and `Coturn <https://github.com/coturn/coturn/wiki>`_ for STUN and TURN servers for your Mattermost installation.
+Pre-requisites
+~~~~~~~~~~~~~~~
 
-Starting a video call
+- Install Docker using the `Ubuntu online guide <https://docs.docker.com/installation/ubuntulinux/`_. Docker installation, configuration and management are out of scope for this guide.
+- Install a Mattermost server using the `official install guides <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_. Mattermost server installation, configuration and management are out of scope for this guide.
+- Install Janus server version 0.2.2.
+- Add an SSL certificate for host **dockerhost**, valid until January 2, 2018.
+- Ability to connect using SSL or plain WebSocket and HTTP // XXX Is this for WebRTC connection?
+
+STUN and TURN servers are not required for this setup.
+
+Deploy Mattermost WebRTC Docker container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After installing Docker, run the following command in a terminal to install the Mattermost WebRTC Docker image.
+
+   .. code:: bash
+
+       docker run --name mattermost-webrtc -p 7088:7088 -p 7089:7089 -p 8188:8188 -p 8189:8189 -d mattermost/webrtc:latest
+
+The command downloads, installs and runs your ``mattermost-webrtc`` container with the Janus Gateway pre-configured to use WebRTC on Chrome, Firefox or the Mattermost Destkop Apps.
+
+.. note::
+  Make sure your Mattermost server can reach the running Mattermost WebRTC Docker container. // XXX How can they verify this?
+
+Configure Mattermost to enable WebRTC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1 - (Optional) Only required if you want to establish the connection to the Mattermost WebRTC Docker container, running the Janus Gateway, with SSL.
+
+Mattermost makes an HTTP request to the Janus Gateway service to get a ``Token``, which is used to identify a user and establish a connection between peers. If you configure Mattermost to make requests to that service with SSL, the SSL certificate used to run the Janus Gateway will not be considered signed by a trusted CA.
```

Comment:
Make sure that is understood that this means the SSL certificate included in the docker image